SPECS/xapi.spec

```diff
@@ -1358,6 +1365,44 @@ Coverage files from unit tests
 %{?_cov_results_package}
 
 %changelog
+* Fri Jun 06 2025 Bengang Yuan <bengang.yuan@cloud.com> - 25.21.0-2
+- Bump release and rebuild
+
+* Fri Jun 06 2025 Bengang Yuan <bengang.yuan@cloud.com> - 25.21.0-1
+- CP-53477 Update host/pool datamodel to support SSH status query and configure
+- CP-53802: Restore SSH service to default state in pool eject
+- CP-53711: Apply SSH settings in joiner before update_non_vm_metadata
```

Comment:
Do we know how the SSH is being disabled? I don't think it's xapi. I don't think this feature is what xcp-ng wants, but if the feature is enabled, XO need to be aware

Reply:
as far as I can tell, it is xapi that disables it, but currently SSH is enabled by default (`default_ssh_enabled` in `xapi-consts`)

It is also not clear to me what the currently in-progress Phase 3 of the SSH feature going to change, so I don't think it's safe to expose to users yet

Reply:
When we heard about that move IIRC we were quite OK about it, as long as there is way to activate it.  That sure looks like something that has to be released in XO before we can roll it out.